app/models/job.rb

```diff
@@ -32,4 +30,10 @@ def self.total_pages(num_of_items_per_page)
     calculation = last_two_months.count / num_of_items_per_page.to_f
     calculation.ceil
   end
+
+  def self.current_openings_technology_count
+    Technology.all.map do |tech|
```

Comment:
@robbiejaeger - Check out batching as an option: http://api.rubyonrails.org/classes/ActiveRecord/Batches.html

Not a huge deal here because of how small Technology is, but `.map` is super inefficient for large records.